### PR TITLE
Don't forward for osde2e orgs or new noalerts label

### PR DESCRIPTION
**Pull request by jewzaam**
https://issues.redhat.com/browse/OSD-5030

### CODE REVIEW

**Comment by jewzaam** [comment]
/hold

Need OSD-4817 to merge first.

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 1]
```diff
@@ -255,11 +257,12 @@ objects:
...
   spec:
     clusterDeploymentSelector:
-      matchLabels:
-        api.openshift.com/managed: "true"
-        api.openshift.com/environment: production
       matchExpressions:
+        - { key: api.openshift.com/managed, operator: In, values: ["true"] }
+        - { key: api.openshift.com/environment, operator: In, values: ["production"] }
         - { key: api.openshift.com/noalerts, operator: NotIn, values: ["true"] }
+        - { key: ext-managed.openshift.io/noalerts, operator: NotIn, values: ["true"] }
```
Comment (jaybeeunix) [inline hack/olm-registry/olm-artifacts-template.yaml:264]: What's this label and how's it different from the noalerts above it?

Comment (jewzaam) [reply hack/olm-registry/olm-artifacts-template.yaml:264]: Before we had a CRD for PD we only had noalerts for excluding a cluster from sending things to PD or DMS.  We asked SDA to create a way for cicd team to add a label to CD's so we could remove our hack, a cronjob on master nodes.  A general solution was put in place that "external" labels could be set via api.  So we support both these labels now in PD and DMS, should for splunk too.

Comment (jaybeeunix) [reply hack/olm-registry/olm-artifacts-template.yaml:264]: ack, thx for the context

**Review by gsleeman — COMMENTED** [review]
/lgtm

**Pull request merged**